src/index.js

```diff
@@ -0,0 +1,19 @@
+import HomeView from './views/HomeView.vue'
+import ForgotPasswordView from './views/ForgotPasswordView.vue'
+import SetAndReSetPasswordView from './views/SetAndReSetPasswordView.vue'
+import AdminLogin from './components/AdminLogin.vue'
+import MeView from './views/MeView.vue'
+import RedirectToLoginMessage from './views/RedirectToLoginMessage.vue'
+import ErrorMessage from './components/ErrorMessage.vue'
+import English from './locales/en.json'
+
+export {
```

Comment:
The exported component names seem to be a little bit generic. HomeView and MeView can be similarly named in accounts, but that is true for ForgotPasswordView and others as well.

Also, why do we return the locale?